src/sysext/sysext.c

```diff
@@ -144,6 +151,92 @@ static int is_our_mount_point(const char *p) {
         return true;
 }
 
+static int need_reload(void) {
+        /* Parse the mounted images to find out if we need
+        to reload the daemon. */
+        int r;
+
```

Comment:
please add this check here:

```c
if (arg_no_reload)
        return false;
```

and then remove any checks for `arg_no_reload` elsewhere, so that we only have a single place where we check this